media/js/firefox/features/tips/tips-youtube.js

```diff
@@ -54,7 +53,12 @@ window.onYouTubeIframeAPIReady = function () {
         );
         var videoId = videoLink.getAttribute('data-id');
 
-        player = new YT.Player(videoLink, {
+        // if youtube API fails or is blocked, try redirecting to youtube.com
+        if (typeof window.YT !== 'undefined') {
+            window.location.href = 'https://www.youtube.com/watch?v=' + videoId;
```

Comment:
Wondering if we could use `window.history.pushState({}, '', window.location);` before the redirect here so the user can click the back button to easily return to this mozilla page. In testing, the back button option either wasn't available once I reached youtube, or it took me back to the page I was on before Tips.

Reply:
This behavior seems to differ depending on the browser, so I'm not sure if `pushState` would introduce other issues here. It may also be something speficic to `localhost` perhaps.